### DISCUSSION TITLE
CAJOE Radiation D v1.1 HV PSU and compatible alpha tubes

### DISCUSSION THREAD

**Discussion by Mr-Blinky** [steadramon/ESPGeiger#18] [General]
I was reading an issue that was posted (https://github.com/steadramon/ESPGeiger/issues/16) and there seemed a little confusion about compatibility with alpha tubes. I can clear up a couple of points regarding this.

The CAJOE Radiation D v1.1 has a variable HV power supply. The voltage is controlled by way of R100. I have just checked with one of my Radiation D v1.1's and can confirm the voltage will range (on my unit) from ~240v to ~625v. This voltage range will suit many alpha GM tubes such as:
Mullard MX-123 (600v - see note below, 2.7 MOhm)
LND712 (500v, 10 MOhm)
Philips 18504/Mullard ZP1400 (500v, 10 MOhm)
SBT-10A/SBT-11A (390v, 30 MOhm per element (from testing))
SI8B (Pankake 390v 5.1 MOhm)

Basically, any GM tube that works within the voltage range of the counter's HV power supply, will work.

There is no drop-in alpha tube that will simply plug into the existing connectors and simply work out of the box. The counter would have to be modified with some kind of flying leads to connect the alpha tube and care should be taken as to the anode resistor value - R33 (I think). This will have to be modified on the counter itself, dependant on tube. The fitted anode resistor is 10 MOhm.

When adjusting a counter's high voltage it is very important to have a high impedance voltmeter for measurement as a standard 20 MOhm meter will pull the HV down by a lot, completely distorting the measurement. If using a regular multimeter (around 20 MOhm) a 1000:1 (1 GOhm : 1 MOhm) voltage divider works very well.

Note: When the HV was adjusted to above 600v the counter started exhibiting strangeness. Phantom clicks and a strange capacitive effect where the counter would go nuts and start screeching when I placed my hand close to the board, without touching it. I would advise to use a voltage of no more than 550v on this unit for reliability.

My closing opinion: I wouldn't use the CAJOE Radiation D v1.1 with an alpha tube. The counter is of low quality and an alpha tube deserves better. I have had a Radiation D v1.1 fail after only a few hundred hours of operation. The fault was with R100 (HV pot) and it dropped the voltage to minimum. Simply turning the pot fixed the issue and was probably dirt on the carbon or wiper. This does support my opinion it is a cheap counter, using cheap parts and a quite rudimentary circuit design. This counter is more suited for hobby experimentation and education, or personal background radiation monitoring.

As alpha tubes are not cheap it would be a waste to throw the readings into doubt, so IMHO a better counter would be more appropriate for an alpha tube. Something along the lines of Net IO GC10 (original or 'next') or even the ESPGieger HW (when released) as I have tested the LND712, Philips 18504 and SI8B tubes with that counter and together they work well. That said, both suggested counters will require the same modifications that the CAJOE Radiation D v1.1 would need.

**Comment by steadramon** [comment]
Thanks for the great information as always @Mr-Blinky
